components/datatypes/commerce.schema.json

```diff
@@ -62,6 +62,114 @@
         "xdm:inStorePurchase": {
           "$ref": "https://ns.adobe.com/xdm/data/measure",
           "description": "'inStore' purchase is saved for analytics use."
+        },
+        "xdm:oneClickPay": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "The number of times the visitor used the single click to pay method to checkout."
+        },
+        "xdm:quickView": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "The number of times a product was selected for quick viewing in a modal or other scaled down product view."
+        },
+        "xdm:orderReview": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Counted when a customer views their order review screen before completing their order."
+        },
+        "xdm:saveForLaterAdd": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Tracks when a visitor adds a product or item to their save for later list."
+        },
+        "xdm:saveForLaterView ": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Tracks when a visitor removes a product or item to their save for later list."
+        },
+        "xdm:saveForLaterRemove": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Tracks when a visitor views their save for later list."
+        },
+        "xdm:checkout": {
```

Comment:
We define nested properties as
"xdm:checkout" {
 "type": "object",
  "properties": {
       All the nested properties are defined here.
  }
}